core/src/main/java/org/apache/carbondata/core/statusmanager/SegmentUpdateStatusManager.java

```diff
@@ -450,8 +450,7 @@ public boolean accept(CarbonFile pathName) {
             String fileName = pathName.getName();
             if (fileName.endsWith(CarbonCommonConstants.DELETE_DELTA_FILE_EXT)
                 && pathName.getSize() > 0) {
```

Comment:
if ( pathName.getSize() > 0 && fileName.endsWith(CarbonCommonConstants.DELETE_DELTA_FILE_EXT))  
will be better

Reply:
@marchpure OK